handler/authentication_handler.go

```diff
@@ -28,28 +35,65 @@ func LoginHandler(c *gin.Context) {
 		c.JSON(http.StatusBadRequest, CreateResponse(fmt.Sprintf("%v", err)))
 		return
 	}
+	// get compnay name
+	providerDetails, err := provider.ProviderControllerObj.GetProvider(credentials.Email)
+	if err != nil {
+		logrus.WithField("err", err).Error("error getting provider")
+		c.JSON(http.StatusBadRequest, CreateResponse(fmt.Sprintf("%v", err)))
+		return
+	}
+	resp.CompanyName = providerDetails.CompanyName
```

Comment:
your approach will have companyName as a different parameter. 

![image](https://github.com/NUS-EVCHARGE/ev-provider-service/assets/10574652/a0c39a98-8333-4a37-97cd-c631149d01a2)

Reply:
if needed might need a way to included within jwt